.github/workflows/nvidia-test.yml

```diff
@@ -33,7 +33,8 @@ jobs:
             - name: Get the artifact
               uses: dawidd6/action-download-artifact@v6
               with:
-                workflow: build.yml
+                workflow_search: true
```

Comment:
```suggestion
```
Is a search needed when the action receives the unique run id?

https://github.com/dawidd6/action-download-artifact/blob/db9477a3ebd885bf3bb8da6a98eeb0d6da2d0d9a/main.js#L61